variation-commons-beacon/src/main/java/uk/ac/ebi/eva/commons/beacon/models/BeaconError.java

```diff
@@ -19,6 +19,10 @@
 
 package uk.ac.ebi.eva.commons.beacon.models;
 
+/*
+ * 	Beacon-specific error. This should be non-null in exceptional situations only,
```

Comment:
Remove the indentation, and wrap exists in quotes like in the description from the specification, because right now it is not clear that it refers to a key.

Reply:
fixed :+1: